windows-installer.iss

```diff
@@ -0,0 +1,92 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Encfs"
+#define MyAppVersion "1.9.5"
+#define MyAppPublisher "encfs"
+#define MyAppURL "https://github.com/vgough/encfs"
+#define MyAppExeName "encfs.exe"
+
+; This variable should contain a path to where libraries needed by encfs are installed.
+#define MyLibraryPath "C:\cygwin\usr\bin"

```

Comment:
Perhaps drop the `usr` part of the path ?
As the real Windows path is `C:\cygwin\bin`, `C:\cygwin\usr\bin` being a mount point done by Cygwin to `C:\cygwin\bin`.

Reply:
Done on both recommendations.

Reply:
Thank you 👍